README.md

```diff
@@ -60,12 +60,19 @@ Alternatively, you can manually edit Application.xml and then reload the Wowza a
   </Property>
 ```
 
-### To deploy plugin code to VM:
-
-- (ensure Wowza application on VM has the properties above)
-- (update version file)
-- (tag version on github)
-- (trigger jenkins build that creates versioned deployment jar artifact)
+### To deploy a new version of the plugin code to the VM:
+
+- Ensure the Wowza application on the VM has the properties mentioned above.
```

Comment:
i think "modules and properties" would be a fine edit if we keep the line.  otherwise, as you point out, deployment and configuration can each happen separately, and the deployment instructions are above, so that'd seem to argue for just removing the line.